style_text/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>How to Style Text Elements with Font, Size & Color in CSS</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Public+Sans:wght@400;700&family=Quicksand:wght@600;700&display=swap" rel="stylesheet"> 
+</head>
+<body>
+
+    <a href="../index.html">&leftarrow; Homeward Bound II: Lost in San Francisco </a>
+
+    <h1>Biscuit topping chocolate bar gummies</h1>
+        <p>
```

Comment:
I would scoot these `<p>` tags one tab to the left, since they aren't direct children of anything except for `<body>`.

If we want to make it clear that they are meant to belong under the heading tags, like `<h1>`. I would do that with blank likes, like so:

```html
    <h1>Stuff</h1>
    <p>Long paragraph full of words...</p>

    <h2>Secondary Stuff</h2>
    <p>Another paragraph below the h2...</p>
```